frontend/src/authentication/hooks/useToken.js

```diff
@@ -45,5 +51,5 @@ export const useToken = () => {
       );
```

Comment:
In the if (!token ) should it be instead if ( !token and !accessToken ) ?

Reply:
updated only using access token now